maps/aurora/aurora-4_mainlevel.dmm

```diff
@@ -48125,6 +48129,11 @@
 	c_tag = "Central Corridor - Camera 2";
 	dir = 8
 	},
+/obj/machinery/alarm{
```

Comment:
Please use the alarm presets instead of setting offsets manually. If your branch does not have the presets, update it.

Reply:
Oh wait, that's fire alarms. Nevermind.